### DISCUSSION TITLE
PrusaSlicer "The configured Post-processing script does not exist"

### DISCUSSION THREAD

**Discussion by ArronRigden** [FormerLurker/ArcWelderLib#87] [Q&A]
When I go to save the G-code in PrusaSlicer v.2.4.0-bata4 I get the error message, "The configured Post-processing script does not exist". I have entered the file location in the output, post-processing scripts (D:\Programs\Arc Welder\bin\ArcWelder.exe). I'm not sure what I'm missing or doing wrong?

**Comment by PeterH1500** [comment]
There's a space character in the address you've used. Put the whole thing in quotes and it'll work fine - 
"D:\Programs\Arc Welder\bin\ArcWelder.exe"

**Reply by ArronRigden** [reply]
PeterH1500, Thank you, that is exactly what I missed. You're the best!!!
